tools/Microsoft.Health.Extensions.BuildTimeCodeGenerator/Microsoft.Health.Extensions.BuildTimeCodeGenerator.targets

```diff
@@ -2,22 +2,22 @@
 
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-    <PropertyGroup>
-        <GeneratorPath>$(MSBuildThisFileDirectory)../../tools/$(TargetFramework)/Microsoft.Health.Extensions.BuildTimeCodeGenerator.dll</GeneratorPath>
-    </PropertyGroup>
+  <PropertyGroup>
+    <GeneratorPath>$(MSBuildThisFileDirectory)../../tools/net6.0/Microsoft.Health.Extensions.BuildTimeCodeGenerator.dll</GeneratorPath>
```

Comment:
We don't necessarily want the user's `TargetFramework` to change our tool's location. A user with the latest dotnet SDK can run `net6.0` applications via `dotnet <dll>`. Furthermore, once `net7.0` releases, this should still work even in `net7.0` projects.